scripts/ci-push.sh

```diff
@@ -4,11 +4,17 @@ set -o errexit -o pipefail
 
 source ./scripts/ci-login.sh
 
-./scripts/build-site.sh
-./scripts/sync-and-test-bucket.sh
 
-# Wait for in-progress jobs to complete before proceeding.
-node ./scripts/await-in-progress.js
-
-./scripts/run-pulumi.sh update
+if [[ "$GITHUB_WORKFLOW" == "Build and deploy" ]]; then
+    # Wait for in-progress jobs to complete before proceeding.
+    ./scripts/build-site.sh
+    ./scripts/sync-and-test-bucket.sh
+    node ./scripts/await-in-progress.js
+    ./scripts/run-pulumi.sh update
+elif [[ "$GITHUB_WORKFLOW" == "Build and deploy new account" ]]; then
+    # temporarily setting site to build in preview mode, to prevent search bots from indexing
+    # until we cut over DNS to start routing to the new account.
+    ./scripts/build-site.sh preview
+    ./scripts/sync-and-test-bucket.sh
+fi
```

Comment:
same `else` to bail if it doesn't understand the workflow.

Reply:
thanks @kmosher, good call on adding an else to bail out.